dashboard/test/models/pd/application/teacher_application_test.rb

```diff
@@ -548,14 +548,12 @@ class TeacherApplicationTest < ActiveSupport::TestCase
         csd_which_grades: ['6'],
         enough_course_hours: options[:enough_course_hours].first,
         previous_yearlong_cdo_pd: ['CS Principles'],
-        replace_existing: options[:replace_existing].second,
```

Comment:
One refactor idea: I don't think it's great that we have to update all of these every time there's a change to the form –– do you think we could extract these as constants up top?

Reply:
Good idea! I'll look into this